lib/fog/compute/google/models/images.rb

```diff
@@ -8,73 +8,72 @@ class Images < Fog::Collection
         # charged a license fee beyond the base Google Compute Engine VM
         # charges. See https://cloud.google.com/compute/docs/operating-systems/
         # for more info.
-        GLOBAL_PROJECTS = [
-          "centos-cloud",
-          "coreos-cloud",
-          "debian-cloud",
-          "google-containers",
-          "opensuse-cloud",
-          "rhel-cloud",
-          "suse-cloud",
-          "ubuntu-os-cloud",
-          "windows-cloud"
-        ].freeze
+        GLOBAL_PROJECTS = %w(
```

Comment:
Awesome. Thanks for updating this. One thing I've wanted for a long time is an API call that provides these projects.

Reply:
Yeah, that would be nice. I can look into if we can get a method for it or if it's just poorly documented.